### DISCUSSION TITLE
Restructure Surface Plant to make it Object Oriented

### DISCUSSION THREAD

**Discussion by malcolm-dsider** [NatLabRockies/GEOPHIRES-X#78] [Ideas]
+Could we restructure this code to split it out into simpler and more logical objects, parents, children, etc? Right now, it is pretty opaque. Given the simple if/then/else structure, this should be too hard. @malcolm-dsider

**Comment by softwareengineerprogrammer** [comment]
This was addressed by https://github.com/NREL/GEOPHIRES-X/pull/96.
